joomla/joomla-cms
# Add parameter for hide/show the article info block title

## What this PR does

This PR add a parameter to hide of show the title of the Article info block. I'm talking about the title <i>Details</i> above the written by, category etc.
I made this PR because it was requested in the code, and also it seems a useful feature.
## How to test this PR

1) Go to an article on the frontend-end and make sure the info block is enabled
2) Enable this PR
3) Confirm on the following places the parameter <i>Show Article info title</i> is visible and works:
- Article options
- Single article menu item options
- Category Blog menu items options
- Article manager configuration

4) Also confirm this parameter works with various combinations of parameters. Like <i>Use default</i> in the article configurtion and <i>Show</i> in the singel articl eoptions.

If I forgot a please where this parameter also can be use, please say so :)
